spec/controllers/users_controller_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'spec_helper'
```

Comment:
Probably out of scope for this PR, but it might be nice to use request specs rather than controller specs for testing the JSON API.